edx_rest_framework_extensions/tests/test_jwt_decoder.py

```diff
@@ -28,29 +28,40 @@ def generate_jwt_token(payload, signing_key=None):
     return jwt.encode(payload, signing_key).decode('utf-8')
 
 
-def generate_jwt_payload(user, scopes=None, filters=None, version='1.0.0', is_restricted=None):
+def generate_jwt_payload(user, scopes=None, filters=None, version=None, is_restricted=None):
     """
     Generate a valid JWT payload given a user and optionally scopes and filters.
     """
+    payload = generate_starting_version(user)
+    payload.update({
+        'version': '1.1.0',
```

Comment:
The constant is intentionally not used here since we want people to manually update this and the expected payload when the version number changes.  I've added a comment to this effect.

Reply:
Might want an assertion to test this hard-coded value against the constant to force the test to break, rather than relying on a comment.